### PR TITLE
Added TGF_PATH environment variable

### PR DESCRIPTION
**Pull request by ycngadeu-coveo**

### CODE REVIEW

**Comment by ycngadeu-coveo** [comment]
I don't understand this error, can anyone maybe explain ?
`verifying github.com/coveo/kingpin/v2@v2.3.0: checksum mismatch`
When are checksums generated ?

**Comment by jocgir** [comment]
I merged the master into your branch. It is always preferable to do it before submitting a pull request.

File: get-latest-tgf.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 get_local_tgf_version () {
-    TGF_LOCAL_VERSION=$(tgf --current-version | awk -F\  '{print $2}')
+    TGF_LOCAL_VERSION=$($TGF_PATH/tgf --current-version | awk -F\  '{print $2}')
```
Comment (julienduchesne) [inline get-latest-tgf.sh]: TGF_PATH is not set here. This causes the script to update TGF everytime

Comment (julienduchesne) [reply get-latest-tgf.sh]: I would set TGF_PATH at the beginning completely (right after the shebang)

File: get-latest-tgf.sh [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
```
Comment (jocgir) [inline get-latest-tgf.sh]: You can init a variable here like that

```suggestion
#!/usr/bin/env bash

TGF=${TGF_PATH:=/usr/local/bin}/tgf

```

File: get-latest-tgf.sh (hunk 1 above)
Comment (jocgir) [reply get-latest-tgf.sh]:
```suggestion
    TGF_LOCAL_VERSION=$($TGF --current-version | awk -F\  '{print $2}')
```

File: get-latest-tgf.sh [hunk 3]
```diff
@@ -20,21 +20,21 @@ get_latest_tgf_version () {
 
 script_end () {
     echo 'Done.'
-    tgf --current-version
+    $TGF_PATH/tgf --current-version
```
Comment (jocgir) [inline get-latest-tgf.sh]:
```suggestion
    $TGF --current-version
```

File: get-latest-tgf.sh [hunk 4]
```diff
@@ -20,21 +20,21 @@ get_latest_tgf_version () {
...
     exit 0
 }
 
 install_latest_tgf () {
     VERSION=$(echo $TGF_LATEST_VERSION | cut -d'v' -f 2)
-    
+    TGF_PATH=${TGF_PATH:-/usr/local/bin}
```
Comment (jocgir) [inline get-latest-tgf.sh]: This line could be deleted.
```suggestion
```

Comment (ycngadeu-coveo) [reply get-latest-tgf.sh]: It kinda breaks stuff if I delete it because the default path is not set anywhere else

Comment (jocgir) [reply get-latest-tgf.sh]: No, it's ok. The first line set it `:=` instead of `:-` does the job.

Comment (ycngadeu-coveo) [reply get-latest-tgf.sh]: So you're saying I could move that line at the top and replace `:=` with `:-` ?

Comment (jocgir) [reply get-latest-tgf.sh]: I already suggested that change.

**Pull request merged**